test/tests/tsdx-build-closure-compiler.test.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @jest-environment node
+ */
+
+const shell = require('shelljs');
+const util = require('../fixtures/util');
+
+shell.config.silent = false;
+
+const stageName = 'stage-build-closure-compiler';
+
+const cjsBundleDev = 'dist/build-withconfig.cjs.development.js';
+const cjsBundleProd = 'dist/build-withconfig.cjs.production.min.js';
+const esmBundle = 'dist/build-withconfig.esm.js';
```

Comment:
think these should be left in the code as literals for now to be consistent with all the other tests

Reply:
I'm probably gonna make some test helpers to cut down on use of literals soon; I've been meaning to, and they'd make renames simpler, among other consistency